back/index.js

```diff
@@ -213,6 +212,24 @@ app.route('/feed').get(verifyToken, (request, response) => {
 	)
 })
 
+app.route('/posts/:id').get(verifyToken, (request, response) => {
```

Comment:
looks more like a `/users/:id/posts`: get the posts of an identified user (aka profile)